matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -1008,31 +856,10 @@ internal class DefaultCryptoService @Inject constructor(
      * @param warn true to warn when some unknown devices are detected.
      */
     override fun setWarnOnUnknownDevices(warn: Boolean) {
+        // TODO this doesn't seem to be used anymore?
```

Comment:
This has been imported from the legacy SDK, still waiting to be plugged, but maybe we can just remove it for now.